packages/core/src/fhir-to-cda/cda-templates/components/__tests__/xmls/encounters-section-single-entry.xml

```diff
@@ -94,6 +107,38 @@
 				</assignedPerson>
 			</assignedEntity>
 		</performer>
+		<participant typeCode="LOC">
+			<participantRole classCode="SDLOC">
+				<id root="placeholder-ORG-OID" extension="${locationId}">
+				</id>
```

Comment:
${locationId} looks like something that was supposed to have a real value?

Reply:
no, these are placeholders that are then replaced by actual values during testing